04week/nasa/css/reset.css

```diff
@@ -0,0 +1,20 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
you should not have previous commits in this branch! make sure you're making a new branch from gh-pages each time